plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: plugins/katello/documentation
+title: Use Katello with Remote Databases
+version: nightly
+---
+
+# Setup Katello with remote databases
+
+Katello can be installed with remote databases for both postgresql and mongo.
+
+## High level
+There are two ways to deploy Katello with remote databases:
+
+1. [Fresh install](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#fresh-install)
```

Comment:
Should we note here that it is not required to externalize both databases? They can be externalized separately, or only one of them.